tests/stub/iteration.py

```diff
@@ -211,11 +211,11 @@ def test_all(self):
     """
 
     def test_nested(self):
-        if get_driver_name() in ['javascript']:
-            self.skipTest('It could not guarantee the order of records requests between in the nested transactions')
         # ex JAVA - java completely pulls the first query before running the second
-        if get_driver_name() not in ['go', 'dotnet', 'javascript']:
-            self.skipTest("Need support for specifying session fetch size in testkit backend")
+        if get_driver_name() in ["java"]:
+            self.skipTest(
+                "completely pulls the first query before running the second"
```

Comment:
Do we know why it worked previously? Was it because the legacy stub server was hiding some behaviour?

Reply:
I think it didn't work before. `if get_driver_name() not in ['go', 'dotnet', 'javascript']:` disabled the test for Python and Java, admittedly in a somewhat convoluted way.